src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect } from 'react';
+import { Errors } from '../../types/Errors';
+import classNames from 'classnames';
+
+type Props = {
+  errorMessage: string;
```

Comment:
use ur `enum` Errors
```suggestion
  errorMessage: Errors;
```

Reply:
Didn't fixed